tools/cldr-apps/src/main/webapp/js/new/cldrSurvey.js

```diff
@@ -12,14 +12,118 @@
  */
 
 const cldrSurvey = (function () {
-  const SURVEY_DEBUG = true;
-
   /*
    * INHERITANCE_MARKER indicates that the value of a candidate item is inherited.
    * Compare INHERITANCE_MARKER in CldrUtility.java.
    */
   const INHERITANCE_MARKER = "↑↑↑";
 
+  const xpathMap = new XpathMap(); // TODO: is it really a singleton?
```

Comment:
it should be a singleton.

Reply:
Thanks! BTW the way it's initialized here assumes cldrXpathMap.js is loaded before cldrSurvey.js. PR 899 uses lazy initialization instead, with a getter. I'm testing by loading the js files in alphabetical order, and the reverse order, trying to make sure the order doesn't matter. Soon we should be ready for some tools like Webpack to help fix such problems.